### PR TITLE
Java code formatting

### PR DESCRIPTION
**Pull request by kelnos**
* Use [google-java-format](https://github.com/google/google-java-format) to format the generated Java code.  I don't love everything about Google's code style guidelines, but this output is _way_ better than javaparser's not-so-pretty printer.
* Sort class body declarations in a sane way so things look nice.

**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowlege that all my contributions will be made under the project's license.

### CODE REVIEW

File: modules/codegen/src/main/scala/com/twilio/guardrail/generators/syntax/Java.scala [hunk 1]
```diff
@@ -177,6 +173,38 @@ object Java {
       }
   }
 
+  def sortDefinitions(defns: List[BodyDeclaration[_]]): List[BodyDeclaration[_]] = {
+    import com.github.javaparser.ast.Modifier._
+    def sortKeyFor(x: BodyDeclaration[_]): Int = x match {
+      case cd: ClassOrInterfaceDeclaration if cd.getModifiers.contains(PUBLIC)                              => 0
+      case cd: ClassOrInterfaceDeclaration if !cd.getModifiers.contains(PRIVATE)                            => 10
```
Comment (blast-hardcheese) [inline modules/codegen/src/main/scala/com/twilio/guardrail/generators/syntax/Java.scala]: `if (a) {} else if (!a) {} else {}`?

Comment (kelnos) [reply modules/codegen/src/main/scala/com/twilio/guardrail/generators/syntax/Java.scala]: This comment does make sense for a few of the cases below, but not this one.  Will clean up the other ones before merging.

Comment (kelnos) [reply modules/codegen/src/main/scala/com/twilio/guardrail/generators/syntax/Java.scala]: (Remember, the order in Java for access is public > protected > (no modifier) > private.)

Comment (blast-hardcheese) [reply modules/codegen/src/main/scala/com/twilio/guardrail/generators/syntax/Java.scala]: Honestly misread this to be `PUBLIC` `!PUBLIC`

**Pull request merged**